### ISSUE TITLE
Skaffold Run Issue

### ISSUE THREAD

**Issue by vishal-biyani** [fission/fission#1562]
The current Skaffold config when used gives below error with the latest version of Skaffold (v1.6.0). There seems a need to upgrade config and fix the error below

```
$ skaffold run
FATA[0000] creating runner: parsing skaffold config: unable to parse config: yaml: unmarshal errors:
  line 32: cannot unmarshal !!map into string
```

**Comment by vishal-biyani** [comment]
The issue is related to nested values in Helm Deploy section which is broken
https://github.com/GoogleContainerTools/skaffold/issues/500

**Comment by vishal-biyani** [comment]
Closed by #1567
